kubernetes/configs/es1-att/commons.sh

```diff
@@ -50,8 +50,8 @@ RSA_KEYS_IDENTIFIERS="490baa58-8d24-4adf-bfb3-f19a9e505b9a"
 SELFCARE_V2_URL="https://api.selfcare.pagopa.it/external/v2"
 
 INTEROP_SELFCARE_PRODUCT_NAME="prod-interop-atst"
-TENANT_ALLOWED_ORIGINS="IPA,ANAC,IVASS,INFOCAMERE,SELC,PDND_INFOCAMERE-SCP"
-PRODUCER_ALLOWED_ORIGINS="IPA,ANAC,IVASS,INFOCAMERE,SELC,PDND_INFOCAMERE-SCP"
+TENANT_ALLOWED_ORIGINS="IPA,ANAC,IVASS,INFOCAMERE,SELC,PDND_INFOCAMERE-SCP,PDND_INFOCAMERE-PRV"
+PRODUCER_ALLOWED_ORIGINS="IPA,ANAC,IVASS,INFOCAMERE,SELC,PDND_INFOCAMERE-SCP,PDND_INFOCAMERE-PRV"
```

Comment:
I think the code does not handle `PDND_INFOCAMERE-PRV` origin when assigning the tenant kind, and risk analysis may be broken for those organizations

Reply:
Ok, good catch.
Let's open an issue to solve this as soon as possible.